dataduct/etl/etl_pipeline.py

```diff
@@ -417,33 +452,21 @@ def create_steps(self, steps_params, is_bootstrap=False):
         """
         input_node = None
         steps = []
-        steps_params = process_steps(steps_params)
-        for step_param in steps_params:
 
-            # Assume that the preceding step is the input if not specified
-            if isinstance(input_node, S3Node) and \
-                    'input_node' not in step_param and \
-                    'input_path' not in step_param:
-                step_param['input_node'] = input_node
-
-            try:
-                step_class = step_param.pop('step_class')
-                step_args = step_class.arguments_processor(self, step_param)
-            except Exception:
-                logger.error('Error creating step with params : %s', step_param)
-                raise
-
-            try:
-                step = step_class(**step_args)
-            except Exception:
-                logger.error('Error creating step of class %s, step_param %s',
-                             str(step_class.__name__), str(step_args))
-                raise
-
-            # Add the step to the pipeline
-            self.add_step(step, is_bootstrap)
-            input_node = step.output
-            steps.append(step)
+        for step_param in steps_params:
+            step_param = process_steps(step_param)
+            step_class = step_param['step_class']
+
+            if step_class.__name__.startswith('Multi'):
```

Comment:
this should be a property of the etl_step class instead of checking the name like this.